docs/azure_arc_jumpstart/azure_arc_k8s/day2/calico/_index.md

```diff
@@ -0,0 +1,127 @@
+weight: 4
+description: >
+---
+
+## Use the Azure Policy Add-on with a custom Azure Policy to verify the calicocloud/calico enterprise components
+
+The following README will guide you on how to use a custom Azure Policy [Azure Policy for Kubernetes](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/policy-for-kubernetes#:~:text=Azure%20Policy%20extends%20Gatekeeper%20v3,Kubernetes%20clusters%20from%20one%20place.) on an Azure Arc connected Kubernetes Service cluster to verify whether Calico network policy is enabled or not.
+
+> **Note: This guide assumes you already deployed an EKS/GKE cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion using [Terraform for eks](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/eks/eks_terraform/) or [Terraform for gke](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/gke/gke_terraform/).**
+
+## Prerequisites
+
+* Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+* [Install or update Azure CLI to version 2.25.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+* As mentioned, this guide starts at the point where you already have a connected EKS/GKE cluster to Azure Arc.
+
+    ![Existing EKS Azure Arc connected cluster](./arcdemo-eks.png)
+
+    ![Existing GKE Azure Arc connected cluster](./arcdemo-gke.png)
+
+* Before installing the Azure Policy Add-on or enabling any of the service features, your subscription must enable the Microsoft.PolicyInsights resource provider and create a role assignment for the cluster service principal. To do that, open [Azure Cloud Shell](https://shell.azure.com/) and run either the Azure CLI or Azure PowerShell command.
+
+    ![Open Azure Cloud Shell](./03.png)
+
+    Azure CLI:
+
+    ```shell
+    az provider register --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Register-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    To verify successful registration, run either the below Azure CLI or Azure PowerShell command.
+
+    Azure CLI:
+
+    ```shell
+    az provider show --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Get-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    ![AzResourceProvider Bash](./04.png)
+
+
+* Azure Policy installed in Arc Connected Cluster
+
+ > by running the the ```kubectl get pods -n gatekeeper-system ``` command, you will notice all pods have been deployed.
+
+![Showing pods deployment](./05.png)
+
+
+## Deploy a customized Azure policy to verify if calicocloud/calico enterprise components enabled in your Azure Arc Kubernetes cluster.
```

Comment:
```suggestion
## Deploy a customized Azure policy to verify if calicocloud/calico enterprise components are enabled in your Azure Arc-enabled Kubernetes cluster.
```

Reply:
"calicocloud/calico enterprise"

Shouldn't this be "Calico Cloud or Calico Enterprise"?

Reply:
You are right, revise my words